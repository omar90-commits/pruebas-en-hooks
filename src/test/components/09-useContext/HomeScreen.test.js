import React from 'react';
import { shallow, mount } from 'enzyme';
import { HomeScreen } from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Pruebas en <HomeScreen />', () => {
	
	const user = {
		name: 'Omar',
		email: 'ov1356272@gmail.com'
	}

	const wrapper = mount( 
		<UserContext.Provider value={{
			user,
		}}>
			<HomeScreen />
		</UserContext.Provider>
	);
	
	test('Debe de mostrarse correctamente', () => {
		
		expect( wrapper ).toMatchSnapshot();
	});
});