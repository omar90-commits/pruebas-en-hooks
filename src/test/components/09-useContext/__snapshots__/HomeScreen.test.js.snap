// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScreen /> Debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Omar",
   "email": "ov1356272@gmail.com"
}
  </pre>
</div>
`;

exports[`Pruebas en <HomeScreen /> Debe de mostrarse correctamente 2`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Omar",
   "email": "ov1356272@gmail.com"
}
  </pre>
</div>
`;
